yuminstall.py

```diff
@@ -165,6 +165,12 @@ def callback(self, what, amount, total, h, user):
             s = to_unicode(_("<b>Installing %(pkgStr)s</b> (%(size)s)\n")) \
                     % {'pkgStr': pkgStr, 'size': size_string(hdr['size'])}
             summary = to_unicode(gettext.ldgettext("redhat-dist", hdr['summary'] or ""))
+
+            # summary should not contain these characters
+            if any(c in summary for c in ('<', '>')):
+                log.debug("Package " + pkgStr + " has an invalid summary: " + summary)
+                summary = ""
```

Comment:
+1 to this. Maybe by the SGML/XML entities so that Gtk just shows them correctly?